### PR TITLE
Integrated RAG pipeline to readme for easy usage on Google Colab

### PR DESCRIPTION
**Pull request by ckurze**
## Summary of the changes / Why this is an improvement

- Added the RAG pipeline example into the README
- Changed the chat API call due to new OpenAI version
- 
## Checklist

 - [ ] Link to issue this PR refers to (if applicable): 
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by ckurze** [comment]
@amotl Could you please have a final review on the changes and help me in resolving the failed checks?

**Comment by ckurze** [comment]
@amotl Could you please have a final review on the changes and help me in resolving the failed checks?

File: topic/machine-learning/llm-langchain/cratedb-vectorstore-rag-openai-sql.ipynb [hunk 1]
```diff
@@ -41,9 +41,10 @@
     "or leverage the fully managed service of [CrateDB Cloud].\n",
     "\n",
     "If you want to test with the very latest developments, you can run CrateDB instance based\n",
```
Comment (amotl) [inline topic/machine-learning/llm-langchain/cratedb-vectorstore-rag-openai-sql.ipynb:43]:
```suggestion
    "If you want to test with the very latest developments, you can run a CrateDB instance based\n",
```

File: topic/machine-learning/llm-langchain/cratedb-vectorstore-rag-openai-sql.ipynb [hunk 2]
```diff
@@ -41,9 +41,10 @@
...
-    "on a nightly build using Docker:\n",
+    "on Docker:\n",
+    "\n",
     "```\n",
-    "docker run --publish 4200:4200 --publish 5432:5432 --pull=always crate/crate:nightly -Cdiscovery.type=single-node\n",
+    "docker run --publish 4200:4200 --publish 5432:5432 --pull=always crate/crate:5.5 -Cdiscovery.type=single-node\n",
```
Comment (amotl) [inline topic/machine-learning/llm-langchain/cratedb-vectorstore-rag-openai-sql.ipynb:47]: Let's just use `crate`, which is effectively "latest stable".

File: topic/machine-learning/llm-langchain/readme.md [hunk 3]
```diff
@@ -40,21 +40,27 @@ and [CrateDB].
...
 
   LangChain also supports managing conversation history in SQL databases. This notebook exercises
   how that works with CrateDB.
 
+- `cratedb-vectorstore-rag-openai-sql.ipynb` [![Open on GitHub](https://img.shields.io/badge/Open%20on-GitHub-lightgray?logo=GitHub)](conversational_memory.ipynb) [![Launch Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/crate/cratedb-examples/main?labpath=topic%2Fmachine-learning%2Fllm-langchain%2Fcratedb-vectorstore-rag-openai-sql.ipynb) [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/crate/cratedb-examples/blob/main/topic/machine-learning/llm-langchain/cratedb-vectorstore-rag-openai-sql.ipynb)
+
+  This example intentially shows how to use the CrateDB Vector Store using SQL. There might be cases where the default parameters of the LangChain integration are not sufficient or you need to crate more advanced SQL queries.
```
Comment (amotl) [inline topic/machine-learning/llm-langchain/readme.md:60]:
```suggestion
  This example intentially shows how to use the CrateDB Vector Store using SQL. There might be cases where the default parameters of the LangChain integration are not sufficient or you need to use more advanced SQL queries.
```

File: topic/machine-learning/llm-langchain/readme.md [hunk 4]
```diff
@@ -40,21 +40,27 @@ and [CrateDB].
...
+
+  The example still uses LangChain components to split a PDF file into chunks, leverage OpenAI to calculate embeddings and to execute the request towards an LLM.
```
Comment (amotl) [inline topic/machine-learning/llm-langchain/readme.md:62]:
```suggestion
  The example still uses LangChain components to split a PDF file into chunks,
  leverages OpenAI to calculate embeddings, and to execute the request towards an LLM.
```

**Pull request merged**